CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.14.7 (2020-12-10)
```

Comment:
it should be a minor version: 2.15.0
because there are breaking changes.
So you need to list the breaking changes as well: no more regular buttons

Reply:
oh true, i forgot that i remove them ahah